src/zenml/cli/utils.py

```diff
@@ -1047,3 +1058,153 @@ def warn_unsupported_non_default_project() -> None:
             "completed in the coming weeks. For the time being it "
             "is recommended to stay within the `default` project."
         )
+
+
+def print_page_info(page: Page[M]):
+    """Print all information pertaining to a page to show the amount of items and pages"""
+    declare(
+        f"Page `({page.page}/{page.total_pages})`, `{page.total}` items "
+        f"found for the applied filters."
+    )
+
+
+F = TypeVar("F", bound=Callable[..., None])
+
+
+def create_filter_help_text(
+    filter_model: Type[FilterBaseModel], field: str
+) -> str:
+    """Create the help text used in the click option help text.
+
+    Args:
+        filter_model: The filter model to use
+        field: The field within that filter model
+
+    Returns:
+        The help text.
+    """
+
+    if filter_model.is_datatime_field(field):
+        return (
+            f"[DATETIME] The following datetime format is supported: "
+            f"'{FILTERING_DATETIME_FORMAT}'. Make sure to keep it in "
+            f"quotation marks. "
+            f"Example: --{field}="
+            f"'{GenericFilterOps.GTE}:{FILTERING_DATETIME_FORMAT}' to "
+            f"filter for everything created on or after the given date."
+        )
+    elif filter_model.is_uuid_field(field):
+        return (
+            f"[UUID] Example: --{field}='{GenericFilterOps.STARTSWITH}:ab53ca' to "
+            f"filter for all UUIDs starting with that prefix."
+        )
+    elif filter_model.is_int_field(field):
+        return (
+            f"[INTEGER] Example: --{field}='{GenericFilterOps.STARTSWITH}:ab53ca' to "
+            f"filter for all UUIDs starting with that prefix."
+        )
+    elif filter_model.is_bool_field(field):
+        return (
+            f"[BOOL] Example: --{field}='{GenericFilterOps.STARTSWITH}:ab53ca' to "
+            f"filter for all UUIDs starting with that prefix."
+        )
+    elif filter_model.is_str_field(field):
+        return (
+            f"[STRING] Example: --{field}='{GenericFilterOps.CONTAINS}:example' to "
+            f"filter everything that contains the query somewhere in its "
+            f"{field}."
+        )
+
+
+def create_data_type_help_text(
```

Comment:
can't we get this automatically somehow without this helper function?

Reply:
The only alternative I saw was mobing this onthe the BaseFilterModel but that feels more wrong, as its a cli concern